### PR TITLE
Aden 3955

### PR DESCRIPTION
**Pull request by jacekjursza**
* fix TestAdsBtfBlocking on oasis
* add timer to track time after gpt loads

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java [hunk 1]
```diff
@@ -377,12 +395,13 @@ public AdsBaseObject waitForPageLoaded() {
...
   public AdsBaseObject waitForPageLoadedWithGpt() {
+    timerStart();
     waitForPageLoaded();
 
     String waitForGPTJS = "typeof window.googletag === 'object'";
     jsActions.waitForJavaScriptTruthy(waitForGPTJS);
-    PageObjectLogging.log("GPT Loaded", String.valueOf(jsActions.execute(waitForGPTJS)), true);
-
+    PageObjectLogging.log("GPT Loaded", "after " +
+                                        String.valueOf(getNumberOfSecondsFromStart()) + " s", true);
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java]: MINOR Directly append the argument of String.valueOf(). [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1153)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Comment by v3rth** [comment]
LGTM

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Pull request merged**